app/components/forms/wizard/basic-details-step.js

```diff
@@ -91,6 +91,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return this.get('data.event.topic.subTopics');
   }),
 
+  showDraftButton: computed('data.event.state', 'data.event.location', function() {
```

Comment:
why do you want to watch `data.event.location`?

Reply:
Accidentally put it there, removing it